aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -59,6 +59,20 @@ SignalR hub methods now support injecting services through dependency injection
 
 Hub constructors can accept services from DI as parameters, which can be stored in properties on the class for use in a hub method. For more information, see [Inject services into a hub](xref:signalr/hubs?view=aspnetcore-7.0&preserve-view=true#inject-services-into-a-hub)
 
+## Performance
+
+### Improved HTTP/2 performance
+
+HTTP/2 allows up to 100 requests to run on a TCP connection in parallel, which is called multiplexing. Previous versions used the [lock](/dotnet/csharp/language-reference/statements/lock) statement to control which request could write to the TCP connection. While lock is a simple solution to writing safe multi-threading code, it’s inefficient under high thread contention. In ASP.NET Core 7, a  [thread-safe queue](https://devblogs.microsoft.com/dotnet/an-introduction-to-system-threading-channels/) replaces the write lock. The tread-safe queue results in a dramatic improvement in the Kestrel gRPC benchmark:
+
+![Entity diagram](https://user-images.githubusercontent.com/219224/177910504-e93579b4-02e4-4079-8a8c-d9d24857aabf.png)
+
+## Server
+
+### New ServerReady event for measuring startup time
+
+The [`ServerReady`](https://github.com/dotnet/aspnetcore/blob/v7.0.0-preview.5.22303.8/src/Hosting/Hosting/src/Internal/HostingEventSource.cs#L75-L79) event has been added to measure [startup time](https://github.com/dotnet/aspnetcore/blob/v7.0.0-preview.5.22303.8/src/Hosting/Hosting/src/GenericHost/GenericWebHostService.cs#L138) of ASP.NET Core apps.
```

Comment:
@adityamandaleeka  provide a suggestion if this is not OK. 
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-7.0?view=aspnetcore-1.0&branch=pr-en-us-26359#server)